File: 백준/Bronze/30045. ZOAC 6/ZOAC 6.cs
int index = int.Parse(Console.ReadLine());

int cnt = 0;

for (int i = 0; i < index; i++)
{
    string str = Console.ReadLine();

    if (str.Contains("01") || str.Contains("OI"))
    {
        cnt++;
    }
}

Console.WriteLine(cnt);

File: 백준/Bronze/30045. ZOAC 6/ZOAC 6.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int index = sc.nextInt();
        int cnt = 0;
        sc.nextLine();
        
        for (int i = 0; i < index; i++) {
            String str = sc.nextLine();
            if (str.contains("01") || str.contains("OI")) {
                cnt++;
            }
        }
        sc.close();
        System.out.println(cnt);
    }
}
